docs/migration/manual-migration.mdx

```diff
@@ -11,8 +11,73 @@ This manual migration process should be run after the
 [automated migration process](./automated-migration), to complete the missing
 parts, or debug issues in the migration CLI output.
 
-:::info WIP
+## SASS Migration
 
-add manual instructions here
+In the latest version of Front-Commerce we converted all sass variables to css
+variables during the [automated migration](./automated-migration). This allows
+for a more modular approach to theming and a better developer experience.
+However, there are still some manual steps that need to be taken to complete the
+migration.
 
-:::
+### Media Mixin
+
+If you make use of the
+[`media-breakpoint-up`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss#L23-34)
+mixin, you will need to directly import the
+[\_breakpoints.scss](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Breakpoints/_breakpoints.scss)
+file at the location where it is used.
+
+If you have the following example component:
+
+```scss title="theme/components/atoms/Example/Example.scss"
+@include media-breakpoint-up(md) {
+  .Example {
+    color: red;
+  }
+}
+```
+
+You would need to import the `_breakpoints.scss` directly in the `Example.scss`
+file
+
+```scss title="theme/components/atoms/Example/Example.scss"
+// add-next-line
+@import "theme/components/atoms/Breakpoints/breakpoints";
+
+@include media-breakpoint-up(md) {
+  .Example {
+    color: red;
+  }
+}
+```
+
+### Color functions
+
+If you use any sass
+[color functions](https://sass-lang.com/documentation/modules/color) you will
+need to directly import the
+[color.scss](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Colors/_colors.scss)
+file at the location where it is used. This is because sass can not process the
+functions on css variables.
```

Comment:
```suggestion
If you use any Sass
[color functions](https://sass-lang.com/documentation/modules/color) you will
need to directly import the
[color.scss](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/01f9c95003f7d4204acd12264c3375464e6c3a12/theme-chocolatine/web/theme/components/atoms/Colors/_colors.scss)
file at the location where it is used. This is because Sass can not process the
functions on CSS variables.
```